fmc/src/error.rs

```diff
@@ -0,0 +1,52 @@
+/*++
+
+Licensed under the Apache-2.0 license.
+
+File Name:
+
+    error.rs
+
+Abstract:
+
+    File contains API and macros used by the FMC for error handling
+
+--*/
+
+/// Caliptra Component
+pub enum FmcComponent {
+    /// Global Error
+    Global = 0x103,
```

Comment:
This might collide with the error space on the ROM?

We need to talk to Vishal during the next ROM standup about error namespace coordination.

Reply:
Let's use 0x203 for FMC and 0x303 for RT.

Reply:
Good hook to have a user guide with low level details for caliptra developers.

Reply:
Error codes changed.